internal/distro/fedora/package_sets.go

```diff
@@ -436,6 +436,47 @@ func iotInstallerPackageSet(t *imageType) rpmmd.PackageSet {
 	return ps
 }
 
+func liveInstallerPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"@workstation-product-environment", // in ks this is @^, what does that mean?
```

Comment:
nitpick: the comment does not provide much value

Reply:
Yea, this question has been answered in the PR (and by @bcl) so the comment goes.